dev/ansible/00_create_intention.yml

```diff
@@ -0,0 +1,29 @@
+- name: set variable
+  set_fact:
+    consul_token: "{{ lookup('env', 'consul_master_token') }}"
+
+- name: Check if intention nifi => nifi_registry (allow) exists
+  shell: consul intention match -source nifi
+  register: nifi_intention
+  when: lookup('env', 'consul_acl') | bool and lookup('env', 'consul_acl_default_policy') == 'deny'
+  environment:
+    CONSUL_HTTP_TOKEN: "{{ consul_token }}"
+
+- name: Consul create intention nifi => nifi_registry (allow)
+  shell: consul intention create nifi nifi_registry
```

Comment:
I do think you have wrong service for nifi-registry? Shouldn’t it be `nifi-registry` ?